package.json

```diff
@@ -8,7 +8,9 @@
     "debug": "node --inspect-brk server.js",
     "test": "npx jest",
     "test:debug": "node --inspect-brk node_modules/.bin/jest --runInBand",
-    "generate-test-resources": "npx generate-odata-client -i social-media-accounts -o integration-tests/social-media-accounts/odata-client -s ./integration-tests/social-media-accounts/service-mapping.json --forceOverwrite"
+    "generate-test-resources:social-media": "npx generate-odata-client -i social-media-accounts -o integration-tests/social-media-accounts/odata-client -s ./integration-tests/social-media-accounts/service-mapping.json --forceOverwrite",
+    "generate-test-resources:timesheet": "npx generate-odata-client -i timeSheetEntryCollection/ -o integration-tests/timeSheetEntryCollection/odata-client -s ./integration-tests/timeSheetEntryCollection/service-mapping.json --forceOverwrite",
```

Comment:
Oooooh, now I get it. I was wondering why we are generating this for the timesheet and not for bupa. Sorry for not mentioning this earlier - use [this npm module](https://www.npmjs.com/package/@sap/cloud-sdk-vdm-workforce-timesheet-service) instead of generating the timesheet service on your own...

Reply:
I deleted the generated services and the app is still working, I guess they were not used.

Reply:
What I mean is - you should not generate this service on your own. But use the official pregenerated npm package.